homeassistant/components/yamaha/media_player.py

```diff
@@ -217,6 +217,7 @@ def update(self) -> None:
             self._play_status = self.receiver.play_status()
         except requests.exceptions.ConnectionError:
             _LOGGER.info("Receiver is offline: %s", self._name)
+            self._attr_state = MediaPlayerState.OFF
```

Comment:
This seems incorrect; there is a connection error, so the state should be "unavailable" (implemented by setting the `available` property).

Reply:
@frenck I only see the connection error when the device is off. Back when I used the "network in standby" mode I never saw this but in an effort to save electricity I want my devices to turn off as much as possible. This becomes a problem when other members of the house turn this receiver off with its remote.

Do you want me to set `self._attr_available` or implement `@property  def available(self)`? Can I still set the state to off? It really helps because certain automations I have will do something else if the device is off.

Reply:
> Do you want me to set self._attr_available or implement @property  def available(self)?

Both result in the same.

> Can I still set the state to off? It really helps because certain automations I have will do something else if the device is off.

No, it will be unavailable. As per code, it is throwing an connection error. A connection error means we can't connect to the device, it doesn't mean we are "off", it could be "on"... we don't know 🤷